### PR TITLE
fix(schema_keys_removal): remove useless keys

### PR DESCRIPTION
**Pull request by jbarreau**

### CODE REVIEW

File: src/agent_toolkit/forestadmin/agent_toolkit/services/permissions/permission_service.py [hunk 1]
```diff
@@ -197,6 +197,5 @@ async def _find_action_from_endpoint(
             action
             for action in actions
             if action["id"] == f"{collection.name}-{get_params['action_name']}-{get_params['slug']}"
-            and http_method.value == action["httpMethod"]
```
Comment (jbarreau) [inline src/agent_toolkit/forestadmin/agent_toolkit/services/permissions/permission_service.py:200]: This line doesn't exists in agent-nodejs; I could set `and http_method.value == "POST"` but I don't get the value of this verification.

File: src/agent_toolkit/forestadmin/agent_toolkit/utils/forest_schema/generator_collection.py [hunk 2]
```diff
@@ -25,16 +25,11 @@ async def build(prefix: str, collection: Union[Collection, CollectionCustomizer]
...
-            "isVirtual": False,
-            "icon": None,
             "isReadOnly": all(
                 [f["type"] == FieldType.COLUMN and f["is_read_only"] for f in collection.schema["fields"].values()]
             ),
-            "integration": None,
             "isSearchable": collection.schema["searchable"],
-            "onlyForRelationships": False,
             "paginationType": "page",
-            "searchField": None,
```
Comment (jbarreau) [inline src/agent_toolkit/forestadmin/agent_toolkit/utils/forest_schema/generator_collection.py:37]: SearchField on collection exists only in python , it's an error

File: src/agent_toolkit/forestadmin/agent_toolkit/utils/forest_schema/type.py [hunk 3]
```diff
@@ -309,23 +307,17 @@ class ForestServerActionField(TypedDict):
     isRequired: bool
     reference: Optional[str]
     type: Union[ColumnAlias, Literal["File"]]
-    widget: Optional[Literal["belongsto select", "file picker"]]
     widgetEdit: Optional[WidgetEditConfiguration]
-    searchValue: Optional[str]
```
Comment (jbarreau) [inline src/agent_toolkit/forestadmin/agent_toolkit/utils/forest_schema/type.py:314]: IMO this `searchValue` is a typo too

**Review by nicolasalexandre9 — APPROVED** [review]
👍

**Review by matthv — APPROVED** [review]

**Review by matthv — APPROVED** [review]

**Review by nicolasalexandre9 — APPROVED** [review]
👍  again !

**Pull request merged**